projects/aca-playwright-shared/package.json

```diff
@@ -1,12 +1,28 @@
 {
-  "name": "@alfresco/aca-playwright-shared",
+  "name": "@alfresco/aca-akash",
   "version": "0.0.1",
+  "commit": "",
   "license": "LGPL-3.0",
-  "dependencies": {
-    "tslib": "^2.0.0"
-  },
-  "publishConfig": {
-    "access": "public"
+  "scripts": {},
+  "peerDependencies": {
+    "@alfresco/adf-content-services": "7.0.0-alpha.7-12408701331",
+    "@alfresco/adf-core": "7.0.0-alpha.7-12408701331",
+    "@alfresco/adf-extensions": "7.0.0-alpha.7-12408701331",
+    "@alfresco/eslint-plugin-eslint-angular": "7.0.0-alpha.7-12408701331",
+    "@alfresco/js-api": "8.0.0-alpha.7-12408701331",
+    "@angular/animations": ">=15.2",
```

Comment:
I don't think you need angular as dependency for playwirght code right?

Reply:
yeah, I removed them